crates/release-automation/src/lib/crate_selection/tests/mod.rs

```diff
@@ -86,7 +86,7 @@ fn release_selection() {
 }
 
 #[test]
-fn members_dependencies() {
```

Comment:
i think i should have not removed this in #2177. i was mistaken and thought you had introduced this test. there might be an opportunity to simplify that test but it doesn't have to be taken now. let me know if you prefer if i make another PR to your PR to bring this one back.

Reply:
https://github.com/holochain/holochain/pull/2178

Reply:
The tests have failed (flake) so I'll push a new commit with this test restored

Reply:
Sorry, page wasn't refreshed, you got to it already. Thank you!